pydart/transpiler.py

```diff
@@ -510,8 +510,9 @@ def visit_unsupported_body(self, name, body):
         return buf
 
     def visit_Raise(self, node):
-        exc = self.visit(node.exc)
-        return f"throw new {exc};"
+        if node.exc is not None:
+            return "throw new {};".format(self.visit(node.exc))
```

Comment:
For self transpilation and consistency, f-strings work better here.

Reply:
We will need to support `str.format` way before we have a working transpiled transpiler.